client/components/npmInfo.js

```diff
@@ -6,16 +6,16 @@ export const NpmInfo = React.createClass({
 	propTypes: {
     	npm: React.PropTypes.object.isRequired
   	},
+
 	render () {
 		return (
 			<div className="panel-module">
```

Comment:
what the beans?! that is really bizarre because it does not look that way in my editor = /